sigstore-java/build.gradle.kts

```diff
@@ -57,6 +57,9 @@ dependencies {
     testImplementation("com.squareup.okhttp3:mockwebserver:4.12.0")
     testImplementation("net.sourceforge.htmlunit:htmlunit:2.70.0")
     testImplementation("org.eclipse.jetty:jetty-server:11.0.20")
+
+    testImplementation("io.github.netmikey.logunit:logunit-core:2.0.0")
+    testRuntimeOnly("io.github.netmikey.logunit:logunit-jul:2.0.0")
```

Comment:
Could you please add `io.github.netmikey.logunit` to `renovate.json` so we get synchronized version bumps by Renovate?